src/freeglut.mk

```diff
@@ -17,16 +17,19 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
-    cd '$(1)' && ./autogen.sh
-    cd '$(1)' && ./configure \
-        $(MXE_CONFIGURE_OPTS) \
-        --enable-replace-glut \
-        --disable-debug \
-        --without-x
-    $(MAKE) -C '$(1)' -j '$(JOBS)' install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS= $(if $(BUILD_STATIC),EXPORT_FLAGS='-DFREEGLUT_STATIC')
+    mkdir '$(1).build'
+    cd    '$(1).build' && $(TARGET)-cmake '$(1)' \
+        -DFREEGLUT_GLES=OFF \
+        -DFREEGLUT_BUILD_DEMOS=OFF \
+        -DFREEGLUT_REPLACE_GLUT=ON \
+        -DFREEGLUT_BUILD_STATIC_LIBS=$(if $(BUILD_STATIC),true,false) \
+        -DFREEGLUT_BUILD_SHARED_LIBS=$(if $(BUILD_STATIC),false,true)
+    $(MAKE) -C '$(1).build' -j '$(JOBS)'
+    $(MAKE) -C '$(1).build' -j 1 install
 
     '$(TARGET)-gcc' \
         -W -Wall -Werror -ansi -pedantic \
         '$(2).c' -o '$(PREFIX)/$(TARGET)/bin/test-freeglut.exe' \
-        `'$(TARGET)-pkg-config' glut --cflags --libs`
+        $(if $(BUILD_STATIC),-DFREEGLUT_STATIC) \
+        -L'$(PREFIX)/$(TARGET)/lib' -lglut -lglu32 -lopengl32 -lgdi32 -lwinmm
```

Comment:
@tonytheodore 
Hmm, after quick look I see that `.pc` file is already generated by cmake. It just has wrong file name. I'll fix this ASAP.